invisible_cities/reco/psf_functions.py

```diff
@@ -0,0 +1,141 @@
+import numpy  as np
+import pandas as pd
+
+from typing import Tuple
+from typing import List
+
+from ..     core.core_functions import shift_to_bin_centers
+from ..     core.core_functions import in_range
+from .. database                import load_db
+
+def create_psf(pos    : Tuple[np.ndarray, ...],
+               charge : np.ndarray,
+               nbins  : int,
+               ranges : List[List[float]]
+               ) -> Tuple[np.ndarray, np.ndarray, List[np.ndarray]] :
+    """
+    Computes the point-spread (PSF) function of a given dataset.
+
+    Parameters
+    ----------
+    pos    : Hits relative position. Only tested for 2D.
+    charge : Hits SiPM charge normalized to the total peak charge.
+    nbins  : The number of bins in each dimension.
+    ranges : Range of the PSF in each dimension.
+
+    Returns
+    ----------
+    psf         : Point-spread function.
+    entries     : Number of entries per bin in the PSF.:
+    bin_centers : Bin centers of the PSF.
+    """
+
+    entries, edges = np.histogramdd(pos, nbins, range=ranges, normed=False)
+    sumC   , edges = np.histogramdd(pos, nbins, range=ranges, normed=False, weights=charge)
```

Comment:
1.- Yeah, the idea is to eventually do it in 3D (and actually it runs, but since the behaviour at all steps is not fully understood yet, for the moment I'm just promoting it as 2D, but wouldn't want to break the 3d option)
2.-  Will do.
3.- No reason at all, probably when testing the code I tried with and without normalization.